llvm/test/Assembler/x86mmx.ll

```diff
@@ -1,9 +0,0 @@
-; RUN: llvm-as < %s | llvm-dis | FileCheck %s
-; RUN: verify-uselistorder %s
-; Basic smoke test for x86_mmx type.
-
-; CHECK: define x86_mmx @sh16
```

Comment:
What will it be after this patch?

Reply:
`x86_mmx` would be translated to `<1 x i64>`, like in the other tests. That is:
```
define <1 x i64> @sh16(<1 x i64> %A) {
  ret <1 x i64> %A
}
```

Reply:
Why we removed it instead of update?

Reply:
The compatibility* tests already cover bitcode-upgrade, and now the IR text "x86_mmx" is simply a textual-IR _parse-time_ alias for <1 x i64>, so this test isn't really testing anything, anymore. 

I expect to remove the textual-IR parse-time alias in subsequent cleanup, too -- the main reason for it right now is just to avoid the noise in test updates at the same time as the real changes.